src/modals/SharePoll.jsx

```diff
@@ -1,16 +1,103 @@
 import React from 'react';
 import { useLocation } from 'react-router-dom';
+import styled from "styled-components";
+import Button from "../components/Button";
+import ShareIcon from 'assets/ShareIcon.png';
 
+const ModalIconField = styled.div`
+  height: 50px;
+  width: 50px;
+  background: #F5DFBD 0% 0% no-repeat padding-box;
+  border-radius: 50%;
+  display: inline-block;
+  align-self: center;
+`;
+
+const ModalHeader = styled.h5`
+  text-align: center;
+  letter-spacing: 0px;
+  color: #000000;
+  opacity: 1;
+  font-weight: normal;
+  font-size: 30px;
+  padding-top: 20px;
+  padding-bottom: 22px;
+`;
+
+const ModalIcon = styled.img`
+  height: 32px;
+  width: 36px;
+  position: relative;
+  left: 7px;
+  top: 9px;
+`;
+
+const ModalContent = styled.p`
+    text-align: left;
+    letter-spacing: 0px;
+    color: #343A40;
+    opacity: 1;
+    font-weight: normal;
+    font-size: 16px;
+    padding-bottom: 56px;
+`;
+
+const ModalFooter = styled.footer`
+    text-align: left;
+    letter-spacing: 0px;
+    color: #343A40;
+    opacity: 1;
+    font-weight: normal;
+    font-size: 14px;
+    padding-top: 56px;
+    padding-bottom: 10px;
+`;
+
+const URLfield = styled.div`
+    background: #FFFFFF 0% 0% no-repeat padding-box;
+    border: 2px solid #E8EBEE;
+    border-radius: 10px;
+    position: relative;
+    padding-top: 6px;
+    padding-bottom: 3px; 
+    padding-left: 20px;
+    left: 5%;
+    width: 90%;
+    opacity: 1;
+    font-size: 18px;
+    letter-spacing: 0px;
+    color: #343A40;
+    opacity: 1;
+`;
```

Comment:
```suggestion
const Content = styled.p`
  color: ${({ theme }) => theme.colors.neutral[700]};
`;
const Footer = styled.footer`
  color: ${({ theme }) => theme.colors.neutral[700]};
  font-size: 14px;
  margin-bottom: 10px;
`;
const URLfield = styled.div`
  padding: 6px 16px;
  margin: 40px auto;
  border: 2px solid ${({ theme }) => theme.colors.neutral[200]};
  border-radius: 10px;
  width: 90%;
  font-size: 18px;
  color: ${({ theme }) => theme.colors.neutral[700]};
`;
```
Ja bym to widział w ten sposób :D